cli/command/completion/functions.go

```diff
@@ -5,6 +5,7 @@ import (
 	"strings"
 
 	"github.com/docker/cli/cli/command/formatter"
+	"github.com/docker/cli/cli/context/store"
 	"github.com/docker/docker/api/types/container"
 	"github.com/docker/docker/api/types/image"
 	"github.com/docker/docker/api/types/network"
```

Comment:
Starting to consider if we should have sub-packages here, otherwise `cli/command/completion` will pull in too many other packages. We can either use `cli/command/completion/xxx` or alternatively, per command (`cli/command/container/completion`). Or .. maybe it should live elsewhere 🤔